### ISSUE TITLE
Allow new `performance` property in webpack config

### ISSUE THREAD

**Issue by huy-nguyen** [js-dxtools/webpack-validator#149]
Per the [release notes](https://github.com/webpack/webpack/releases/tag/v2.1.0-beta.28) for Webpack's latest beta version, there's a new `performance.hints` property in the config to enable/disable warnings about bundle size. Currently, using this config property 

```javascript
module.exports = {
  // other configs
  performance: {
    hints: false
  }
}
```

causes `webpack-validator` to error out:

```
[1] "performance" is not allowed
```

**Comment by kentcdodds** [comment]
Feel free to file a pull request, but please note the note in the README.

**Comment by huy-nguyen** [comment]
Does that mean people won't need `webpack-validator` after switching to webpack 2?

**Comment by kentcdodds** [comment]
Correct.

**Comment by huy-nguyen** [comment]
Closed by #150.
